External/Plugins/ASCompletion/PluginUI.cs

```diff
@@ -673,6 +679,138 @@ private void RefreshView(FileModel aFile)
             }
         }
 
+        private void UpdateTree(FileModel aFile)
+        {
+            try
+            {
+                if (aFile == FileModel.Ignore)
+                    return;
+
+                var classes = new Dictionary<string, ClassModel>();
+                foreach (var classModel in aFile.Classes)
+                    classes[classModel.Name] = classModel;
```

Comment:
In principle it would be possible for a file to declare 2 classes with the same name. Especially in Haxe when you might be copy-pasting declarations in one file before doing some renaming.